proposals/lip-0052.md

```diff
@@ -30,329 +31,345 @@ Therefore, providing a unified module to handle, transfer and modify NFTs is a n
 
+#### transfer
+
+This function is used to transfer ownership of NFTs within one chain.
+
+#### transferCrossChain
+
+This function is used to transfer ownership of NFTs across chains in the Lisk ecosystem.
+
 #### recover
 
-This function should only be called by the Interoperability module to trigger the recovery of NFTs escrowed to terminated chains.
+This function should only be called by the [Interoperability module][lip-0045] to trigger the recovery of NFTs escrowed to terminated chains.
 
 ## Specification
 
-### Constants and Notations
+### Notation and Constants
 
 The following constants are used throughout the document:
 
-| Name                                  | Type    | Value           |
-|---------------------------------------|---------|-----------------|
-| **Interoperability Constants**        |         |                 |
-| `MIN_RETURN_FEE`                      | uint64  | 1000            |
-| **NFT Module Constants**              |         |                 |
-| `MODULE_ID_NFT`                       | uint32  | TBD             |
-| `COMMAND_ID_TRANSFER`                 | uint32  | 0               |
-| `COMMAND_ID_CROSS_CHAIN_TRANSFER`     | uint32  | 1               |
-| `CROSS_CHAIN_COMMAND_ID_TRANSFER`     | uint32  | 0               |
-| `CHAIN_ID_ALIAS_NATIVE`               | uint32  | 0               |
-| `NFT_NOT_LOCKED`                      | uint32  | `MODULE_ID_NFT` |
-| `MAX_BYTE_LENGTH_ATTRIBUTES`          | uint32  | 9 * 1024        |
-| `CCM_STATUS_OK`                       | uint32  | 0               |
-| `CCM_STATUS_NFT_NOT_SUPPORTED`        | uint32  | 64              |
-| `CCM_STATUS_NFT_PROTOCOL_VIOLATION`   | uint32  | 65              |
-| **Store Constants**                   |         |                 |
-| `STORE_PREFIX_NFT`                    | bytes   | 0x00 00         |
-| `STORE_PREFIX_USER`                   | bytes   | 0x80 00         |
-| `STORE_PREFIX_COLLECTION`             | bytes   | 0xc0 00         |
-| `STORE_PREFIX_AVAILABLE_COLLECTION`   | bytes   | 0xd0 00         |
-| `STORE_KEY_LENGTH_NFT`                | uint32  | 16              |
-| **General Constants**                 |         |                 |
-| `EMPTY_BYTES`                         | bytes   | ""              |
-| `ADDRESS_LENGTH`                      | uint32  | 20              |
-
-#### uint32be Function
-
-The function `uint32be(x)` returns the big endian uint32 serialization of an integer `x`, with `0 <= x < 2^32`. This serialization is always 4 bytes long.
+| Name                                | Type   | Value                   |
+| ----------------------------------- | ------ | ----------------------- |
+| **Interoperability Constants**      |        |                         |
+| `MAINCHAIN_ID`                      | bytes  | 0x 00 00 00 00          |
```

Comment:
This is not used anywhere and could be removed. On the other hand, maybe we should use it to handle mainchain NFTs in the future

Reply:
Done!